File: CS/15_Document/InvisibleAllPdfLayers.cs
using Spire.Pdf;
using Spire.Pdf.Graphics.Layer;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace InvisibleAllPdfLayers
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create a new PDF document.
            PdfDocument doc = new PdfDocument();

            //Load the file from disk.
            doc.LoadFromFile(@"..\..\..\..\..\..\Data\Template_Pdf_5.pdf");

            for (int i = 0; i < doc.Layers.Count; i++)
            {
                //Show all the Pdf layers.
                //doc.Layers[i].Visibility = PdfVisibility.On;

                //Set all the Pdf layers invisible.
                doc.Layers[i].Visibility = PdfVisibility.Off;
            }

            String result = "InvisibleAllPdfLayers_out.pdf";

            //Save the document
            doc.SaveToFile(result);
            //Launch the Pdf file
            PDFDocumentViewer(result);
        }

        private void PDFDocumentViewer(string filename)
        {
            try
            {
                System.Diagnostics.Process.Start(filename);
            }
            catch { }
        }
    }
}

File: VB/15_Document/InvisibleAllPdfLayers.vb
Imports Spire.Pdf
Imports Spire.Pdf.Graphics.Layer

Namespace InvisibleAllPdfLayers
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a new PdfDocument object
			Dim doc As New PdfDocument()

			' Load the PDF document from the specified file path
			doc.LoadFromFile("..\..\..\..\..\..\Data\Template_Pdf_5.pdf")

			' Iterate through each layer in the document
			For i As Integer = 0 To doc.Layers.Count - 1

				' Show all the Pdf layers
				' doc.Layers(i).Visibility = PdfVisibility.On;

				' Set all the Pdf layers to be invisible
				doc.Layers(i).Visibility = PdfVisibility.Off

			Next i

			' Specify the output file path
			Dim result As String = "InvisibleAllPdfLayers_out.pdf"

			' Save the modified document to the output file
			doc.SaveToFile(result)

			' Close the PDF document
			doc.Close()

			' Launch the Pdf file
			PDFDocumentViewer(result)
		End Sub

		Private Sub PDFDocumentViewer(ByVal filename As String)
			Try
				Process.Start(filename)
			Catch
			End Try
		End Sub
	End Class
End Namespace
